lib/server.rb

```diff
@@ -34,6 +35,15 @@ def scope_meta_info(scope)
       "at: #{DateTime.parse(@description[scope].meta.modified).strftime("%F %T")})"
     end
 
+    def latest_update(description)
+      description.attributes.keys.map { |scope| description[scope].meta.try(:[], "modified") }
+        .compact.map { |t| Time.parse(t) }.sort.last
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.